docs/reference/ccr/apis/follow/get-follow-info.asciidoc

```diff
@@ -173,3 +174,19 @@ The API returns the following results:
     ]
 }
 --------------------------------------------------
+
+The API returns the following results if the follower index is `paused`:
+
+[source,console-result]
+--------------------------------------------------
+{
+    "follower_indices" : [
+        {
+            "follower_index" : "follower_index",
+            "remote_cluster" : "remote_cluster",
+            "leader_index" : "leader_index",
+            "status" : "paused"
+        }
+    ]
+}
+--------------------------------------------------
```

Comment:
This snippet is failing with "status: expected String [paused] but was String [active]".  I think we need to skip testing on this snippet or run a pause API and rerun the info API before this snippet.